o/org.aspectj/aspectjweaver_ubi_9.3.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash -e
+# -----------------------------------------------------------------------------
+#
+# Package       : aspectjweaver
+# Version       : V1_9_20_1 
+# Source repo   : https://github.com/eclipse/org.aspectj
+# Tested on	: UBI 9.3
+# Language      : Java
+# Travis-Check  : True
+# Script License: Apache License, Version 2 or later
+# Maintainer	: Siddesh Sangodkar <siddesh.sangodkar1@ibm.com>
+
+#
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+# Variables
+REPO=https://github.com/eclipse/org.aspectj
+
+# Default tag for aspectjweaver
+if [ -z "$1" ]; then
+  export VERSION="V1_9_20_1"
+else
+  export VERSION="$1"
+fi
+
+# install tools and dependent packages
+yum update -y
+yum install -y git wget maven
+
+# install java 
+yum install -y java-11-openjdk-devel
+
+
+# Cloning Repo
+git clone $REPO
+cd ./org.aspectj/
+git checkout ${VERSION}
+
+# check maven
+./mvnw -B --version
+
+#Build package
+./mvnw -B --file pom.xml -DskipTests install
+
+# Tests
+cd aspectjweaver/
+../mvnw -B --file pom.xml -Daspectj.tests.verbose=false verify
+
```

Comment:
Need exit codes as per the currency requirement
exit 1 - if build fails
exit 2 - if unit test fails
exit 0 - if all steps are successful.